src/elements/molecules/Carousel/Carousel.example.jsx

```diff
@@ -0,0 +1,268 @@
+        ),
+        options: {
+          padding: '1rem',
+          background: 'path/or/url/to/image(.jpg|.gif|.png|.svg)',
+          brightness: 0.5,
+        }
+      },
+    ```
+*/
+
+import Image from '@atoms/Image/Image';
+import Button from '@atoms/Button/Button';
+import Card from '@molecules/Card/Card';
+import {
+  Carousel,
+  Carousel__container,
+  Carousel__controls,
+  Carousel__slide
+} from './Carousel';
+
+export default [{
+  examples: [
+    {
+      name: 'Default Carousel',
+      description: '',
+      exports: '',
+      notes: '',
+      component: (
+        <Carousel>
+          <Carousel__controls />
+          <Carousel__container style={{ textAlign: 'center' }}>
+            <Carousel__slide>
+              <Card id="carousel-card-01" className="padding--medium">
+                <p>{Utils.ipsum('paragraph', 2)}</p>
+              </Card>
+            </Carousel__slide>
+            <Carousel__slide>
+              <Card id="carousel-card-02" className="padding--medium">
+                <p>{Utils.ipsum('paragraph', 2)}</p>
+              </Card>
+            </Carousel__slide>
+            <Carousel__slide>
+              <Card id="carousel-card-03" className="padding--medium">
+                <p>{Utils.ipsum('paragraph', 2)}</p>
+              </Card>
+            </Carousel__slide>
+            <Carousel__slide>
+              <Card id="carousel-card-04" className="padding--medium">
+                <p>{Utils.ipsum('paragraph', 2)}</p>
+              </Card>
+            </Carousel__slide>
+          </Carousel__container>
+        </Carousel>
+      )
+    }, {
+      name: 'With Autoplay',
+      description: '',
+      exports: '',
+      notes: '',
+      component: (
+        <Carousel autoplay={true}>
+          <Carousel__controls />
+          <Carousel__container style={{ textAlign: 'center' }}>
+            <Carousel__slide>
+              <Card id="carousel-card-05" className="padding--medium">
+                <p>{Utils.ipsum('paragraph', 2)}</p>
+              </Card>
+            </Carousel__slide>
+            <Carousel__slide>
+              <Card id="carousel-card-06" className="padding--medium">
+                <p>{Utils.ipsum('paragraph', 2)}</p>
+              </Card>
+            </Carousel__slide>
+            <Carousel__slide>
+              <Card id="carousel-card-07" className="padding--medium">
+                <p>{Utils.ipsum('paragraph', 2)}</p>
+              </Card>
+            </Carousel__slide>
+            <Carousel__slide>
+              <Card id="carousel-card-08" className="padding--medium">
+                <p>{Utils.ipsum('paragraph', 2)}</p>
+              </Card>
+            </Carousel__slide>
+          </Carousel__container>
+        </Carousel>
+      )
+    }, {
+      name: 'With Autoplay and Custom Delay',
+      description: '',
+      exports: '',
+      notes: '',
+      component: (
+        <Carousel autoplay={true} delay="3500">
```

Comment:
Can we double check that this is taking custom delay value?
It appears to be animating at the same time as the example above it.